server/migrations/1736504593192_encrypt-users.ts

```diff
@@ -0,0 +1,31 @@
+import { MigrationBuilder, ColumnDefinitions } from 'node-pg-migrate';
+
+export const shorthands: ColumnDefinitions | undefined = undefined;
+
+export async function up(pgm: MigrationBuilder): Promise<void> {
+  pgm.sql(`
+    CREATE EXTENSION IF NOT EXISTS pgcrypto;
+  `);
+  pgm.sql(
+    `ALTER TABLE application.user
+      ALTER COLUMN full_name TYPE bytea USING full_name::bytea,
```

Comment:
I think this will cause problems with existing data, if the server assumes the `bytea` data is encrypted. If we can access the key with `process.env.DATABASE_ENCRYPTION_KEY` in this migration script, I think we should `pgp_sym_encrypt` the existing data in the `USING` clause.